Soccar [Client]/src/nl/soccar/ui/fx/controller/LoginFXMLController.java

```diff
@@ -20,28 +24,46 @@
 public class LoginFXMLController implements Initializable {
 
     @FXML
-    private Button btnSelectSportscar;
+    private Button btnLogin;
     @FXML
-    private Button btnSelectPickup;
+    private TextField txtFieldName;
     @FXML
-    private Button btnSelectCasual;
+    private ToggleButton btnSelectCasualCar;
     @FXML
-    private Button btnLogin;
+    private ToggleButton btnSelectPickup;
     @FXML
-    private TextField txtfieldName;
-
+    private ToggleButton btnSelectSportsCar;
+    
+    private final ToggleGroup TOGGLE_GROUP = new ToggleGroup();

```

Comment:
ToggleGroup is met hoofdletters geschreven, dat betekent dat deze constant is.. Maar hij is, echter, niet static. Ook kan de assignment in de initialize(..) methode gebeuren (dat is netter). Graag assignment plaatsing en de benaming veranderen.